sideboard/tests/test_server.py

```diff
@@ -19,10 +18,15 @@
 
 import sideboard.websockets
 from sideboard.lib import log, config, subscribes, notifies, services, cached_property, WebSocket
-from sideboard.tests import service_patcher, config_patcher
+from sideboard.tests import service_patcher, config_patcher, get_available_port
 from sideboard.tests.test_sa import Session
 
 
+available_port = get_available_port()
+config['cherrypy']['server.socket_port'] = available_port
```

Comment:
By the time this code is executed, we'll have already populated ``cherrypy.config`` with what's in our config file, so the configuration will already be living in two places, each of which we need to update.